doc/README.rst

```diff
@@ -3,161 +3,12 @@
-have ``sphinx`` and ``ipython`` installed. `numpydoc
-<https://github.com/numpy/numpydoc>`_ is used to parse the docstrings that
-follow the Numpy Docstring Standard (see above), but you don't need to install
-this because a local copy of ``numpydoc`` is included in the pandas source
-code. `nbsphinx <https://nbsphinx.readthedocs.io/>`_ is used to convert
-Jupyter notebooks. You will need to install it if you intend to modify any of
-the notebooks included in the documentation.
-
-Furthermore, it is recommended to have all `optional dependencies
-<http://pandas.pydata.org/pandas-docs/dev/install.html#optional-dependencies>`_
-installed. This is not needed, but be aware that you will see some error
-messages. Because all the code in the documentation is executed during the doc
-build, the examples using this optional dependencies will generate errors.
-Run ``pd.show_versions()`` to get an overview of the installed version of all
-dependencies.
-
-.. warning::
-
-   Sphinx version >= 1.2.2 or the older 1.1.3 is required.
-
-Building pandas
-^^^^^^^^^^^^^^^
-
-For a step-by-step overview on how to set up your environment, to work with
-the pandas code and git, see `the developer pages
-<http://pandas.pydata.org/developers.html#working-with-the-code>`_.
-When you start to work on some docs, be sure to update your code to the latest
-development version ('master')::
-
-    git fetch upstream
-    git rebase upstream/master
-
-Often it will be necessary to rebuild the C extension after updating::
-
-    python setup.py build_ext --inplace
-
-Building the documentation
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-So how do you build the docs? Navigate to your local folder
-``pandas/doc/`` directory in the console and run::
-
-    python make.py html
-
-And then you can find the html output in the folder ``pandas/doc/build/html/``.
-
-The first time it will take quite a while, because it has to run all the code
-examples in the documentation and build all generated docstring pages.
-In subsequent evocations, sphinx will try to only build the pages that have
-been modified.
-
-If you want to do a full clean build, do::
-
-    python make.py clean
-    python make.py build
-
-
-Starting with 0.13.1 you can tell ``make.py`` to compile only a single section
-of the docs, greatly reducing the turn-around time for checking your changes.
-You will be prompted to delete `.rst` files that aren't required, since the
-last committed version can always be restored from git.
-
-::
-
-    #omit autosummary and API section
-    python make.py clean
-    python make.py --no-api
-
-    # compile the docs with only a single
-    # section, that which is in indexing.rst
-    python make.py clean
-    python make.py --single indexing
-
-For comparison, a full doc build may take 10 minutes. a ``-no-api`` build
-may take 3 minutes and a single section may take 15 seconds.
-
 Where to start?
 ---------------
 
+Please follow the documentation in ``doc/source/contributing.rst`` for a 
```

Comment:
I'd remove the titles, and paragraphs and leave something as simple as:
```
See `doc/source/contributing.rst` in this repo, or the generated html version https://pandas.pydata.org/pandas-docs/stable/contributing.html
```

Not a big difference, but leaving the titles and some info gives the wrong impression that this is a page (that could be extended) instead of a "redirect".